### ISSUE TITLE
Non-BSD license

### ISSUE THREAD

**Issue by thedaemon** [broozar/installDesktopFreeBSD#5]
For a FreeBSD only product, having a GPL 3 license seems off putting. Can you change to BSD style license? Or like your other projects, Apache?

**Comment by broozar** [comment]
I totally agree. If it were not for the Linux Mint-based dark theme, the license would be BSD. Unfortunately, afaik the Minty theme collection is GPL, and my understanding is that the whole project has to be GPL now due to the viral nature of that license.
If you use another dark theme instead of Minty, feel free to consider any of my code here "truly" free. For the next release, I am looking into alternatives for the theme, I just haven't found any that look and feel equally good.

**Comment by okalm** [comment]
Hi,

Having a separate repository for the theme called by a post install script would do the trick may be ? 
The theme repository would be GPL while this one can be BSD.
